run.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash
 
-JAVA_OPTS="-Djavax.net.ssl.trustStore=/app/trusted.jks -Djavax.net.ssl.trustStorePassword=${TRUST_STORE_PASSWORD} -Djavax.net.ssl.trustStoreType=jks"
+JAVA_OPTS="-Djavax.net.ssl.trustStore=/app/trusted.jks \
+           -Djavax.net.ssl.trustStorePassword=${TRUST_STORE_PASSWORD} \
+           -Djavax.net.ssl.trustStoreType=jks \
+           -Dcom.sun.management.jmxremote.local.only=false \
+           -Ddelius.endpoint.url=${DELIUS_ENDPOINT_URL} \
+           -Ddelius.api.username=${DELIUS_API_USERNAME} \
+           -DAPPLICATION_INSIGHTS_IKEY=${APP_INSIGHTS_KEY} \
+           -Djwt.public.key=${JWT_PUBLIC_KEY} \
```

Comment:
spring will automatically convert `JWT_PUBLIC_KEY` into `jwt.public.key` anyway so not needed if set in the environment

Reply:
Same thing here - explicit definition of values passed from environment.  I will canvass a coupe of other opinions to see which people feel most comfortable with, thanks.